keps/sig-apps/0008-sidecarcontainer.md

```diff
@@ -0,0 +1,148 @@
+---
+kep-number: 0008
+title: Sidecar Containers
+authors:
+  - "@joseph-irving"
+owning-sig: sig-apps
+participating-sigs:
+  - sig-apps
+  - sig-node
+reviewers:
+  - "@fejta"
+approvers:
+  - TBD
```

Comment:
I am loathe to sign up for this.  @kubernetes/sig-node-feature-requests  @kubernetes/sig-apps-feature-requests  Someone from sig-node or sig-apps would be best...